### PR TITLE
feat: release 0.1.0

### PR DESCRIPTION
**Pull request by tzssangglass**

### CODE REVIEW

File: runner-dist/apisix-runner-src-dist/pom.xml [hunk 1]
```diff
@@ -63,8 +63,8 @@
                         </goals>
                         <configuration>
                             <target>
-                                <copy file="${project.build.directory}/apisix-runner-src.tgz"
-                                      tofile="${project.basedir}/../../dist/apache-apisix-java-plugin-runner-${project.version}-src.tgz"
+                                <copy file="${project.build.directory}/apisix-java-plugin-runner.tgz"
+                                      tofile="${project.basedir}/../../apisix-java-plugin-runner-0.1.0.tgz"
```
Comment (dmsolr) [inline runner-dist/apisix-runner-src-dist/pom.xml:67]: I recommend replacing the version number with `${project.version}`.
And the version could exclude suffix `-RELEASE`.

Comment (tzssangglass) [reply runner-dist/apisix-runner-src-dist/pom.xml:67]: got

Comment (tzssangglass) [reply runner-dist/apisix-runner-src-dist/pom.xml]: update

File: Makefile [hunk 2]
```diff
@@ -17,18 +17,18 @@
 
 SHELL := /bin/bash -o pipefail
 
-VERSION ?= 0.1.0-SNAPSHOT
-RELEASE_SRC = apache-apisix-java-plugin-runner-${VERSION}-src
+VERSION ?= 0.1.0
+RELEASE_SRC = apisix-java-plugin-runner-${VERSION}
```
Comment (nic-chen) [inline Makefile:21]: why remove `-src`? other projects have it.

Comment (tzssangglass) [reply Makefile:21]: got

Comment (tzssangglass) [reply Makefile]: update

**Review by dmsolr — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**